agent/api/task/task.go

```diff
@@ -536,9 +536,11 @@ func (task *Task) initServiceConnectResources() error {
 			ContainerName: "service-connect",
 		}
 	}
+	// TODO [SC]: End dev/testing purposes
```

Comment:
What are these TODOs? can you describe in more detail? it's not clear what still needs to be done here or why it can't be done now already

Reply:
This is a marker for the END of the section start at 532. These sections need to be deleted after dev/testing and the backend is fully operational.